frontend/src/metabase/css/login.css

```diff
@@ -139,3 +20,34 @@
   margin-top: 1em;
   text-align: center;
 }
+
+#BridgeImg {
+  position: relative;
+  left: -1240px;
+  bottom: -3px;
+}
+
+@media screen and (min-width: 800px) {
+  #BridgeImg {
+    left: -1040px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  #BridgeImg {
+    left: -840px;
+  }
+}
+
+@media screen and (min-width: 1600px) {
+  #BridgeImg {
+    left: -640px;
+  }
+}
+
+@media screen and (min-width: 1920px) {
+  #BridgeImg {
+    width: 100%; /* ensure the image always stretches the full width */
+    left: 0px;
+  }
+}
```

Comment:
I'm not sure I understand the need for media queries, but I'm sure they're fixing something since we definitely had some jank with the old one on bigger screens.

Is the goal to keep the sailboat in frame on smaller screens? Could we get a similar effect with a percentage in the `background-position-x` property?

Reply:
Yeah the idea was to try and adjust the framing a bit depending on the screen size so that some of the more interesting elements like the bridge and boat and lighthouse aren't over to one side. There may be a better way to do it that wouldn't require media queries and if you have any ideas I'm open to them. 👍

Reply:
Fixing the background position might be a more concise way to always show the bridge or boat at the same position, but I guess there's nothing to be gained by replacing the size-by-size positioning. The media queries definitely give you more control.